2/src/index.html

```diff
@@ -0,0 +1,682 @@
+                id="mask1"
+                mask-type="alpha"
+                maskUnits="userSpaceOnUse"
+                x="42"
+                y="0"
+                width="25"
+                height="24"
+              >
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M42.1484 0.000732422H66.1528V24H42.1484V0.000732422Z"
+                  fill="white"
+                />
+              </mask>
+              <g mask="url(#mask1)">
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M54.1528 19.3426C50.0968 19.3426 46.811 16.0567 46.811 11.9996C46.811 7.94801 50.0968 4.65782 54.1528 4.65782C58.2044 4.65782 61.4913 7.94801 61.4913 11.9996C61.4913 16.0567 58.2044 19.3426 54.1528 19.3426ZM54.1528 0.000732422C47.5212 0.000732422 42.1484 5.37346 42.1484 11.9996C42.1484 18.6269 47.5212 24.0007 54.1528 24.0007C60.7801 24.0007 66.1528 18.6269 66.1528 11.9996C66.1528 5.37346 60.7801 0.000732422 54.1528 0.000732422Z"
+                  fill="white"
+                />
+              </g>
+            </svg>
+          </a>
+        </div>
+
+        <div class="flex-grow-1">
+          <div class="hamburger d-none d-md-block" id="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+
+          <nav class="d-flex d-md-none" id="menuWrapper">
+            <ul class="d-flex">
+              <li>
+                <a href="#"> Why Skout </a>
+              </li>
+              <li>
+                <a href="#"> Products </a>
+              </li>
+              <li>
+                <a href="#"> Partners </a>
+              </li>
+              <li>
+                <a href="#"> Company </a>
+              </li>
+              <li class="active">
+                <a href="#"> Tips & Tools </a>
+              </li>
+              <li>
+                <a href="#"> Blog </a>
+              </li>
+            </ul>
+            <div class="d-flex">
+              <button href="#" class="btn btn-light">Get Skout</button>
+              <button href="#" class="btn btn-light">Log In</button>
+            </div>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="heading container-fluid">
+        <div class="container">
+          <h1>Find trouble before trouble finds you.</h1>
+          <div class="container-sm heading-text-wrapper">
+            <p>
+              SKOUT is a cybersecurity company that delivers total protection
+              for total peace of mind. Whether you’re an international
+              conglomerate or a mom-and-pop shop we make cybersecurity easy and
+              accessible because we believe security is universal and peace of
+              mind is worth protecting.
+            </p>
+            <div class="btn-wrapper">
+              <a href="#" class="btn btn-light"> Why SKOUT </a>
```

Comment:
according to your wrapper class , here should be a button. why do you have a link inside?

Reply:
changed